packages/documentation-site/patternfly-docs/content/design-guidelines/content/writing-patternfly-design-guidelines.md

```diff
@@ -1,74 +1,74 @@
 
-## Design guidelines template
-When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must be under one of the following headings.
+To learn how to contribute design guidelines, see our [contribution guide](https://github.com/patternfly/patternfly-org/wiki/Contributing-to-patternfly-org-for-designers).
+
+## Design guidelines sections
+When creating guidelines, information must be presented in the following order. Sections may be omitted, but if present must use the following section headings (H2).
+
+### Elements 
+Display an image of the component containing numbered annotation tags for each major part of the component's visual design. Include specific icons, headers, fields, symbols, and anything else included in the component's structure. Following the image, list all the parts you tagged, aligning with the number displayed in the respective annotation tag. In this list, provide a description of each one. 
 
-### Elements (H2)
-Introduce the component and list the primary parts that make up its structure. This includes specific icons, headers, fields, symbols, or anything else included in the component.
 Example:
 
-<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="479px" />
+<img src="./img/sliderelements.png" alt="A screenshot of the slider component's element section with 8 elements outlined." width="600px" />
 
-### Usage (H2)
+### Usage 
 Explain how the component is used, when to use it, and why it should be used. Include explanations of problems it could help solve and address its advantages over similar components. Include images for visual examples.
 
 Additional sub-sections to include: 
-#### When to use
-#### When not to use
-#### When to use X vs. Y
-#### Examples of incorrect usage
-
+#### When to use (H3)
+#### When not to use (H3)
+#### When to use X vs. Y (H3)
+#### Examples of incorrect usage (H3)
 
-### Behavior (H2)
-Explain how the component functions and break down how its elements interact with each other to achieve its purpose.
+### Behavior 
+Explain how the component functions and break down how its elements interact with each other to achieve its purpose. It may be helpful to include additional images or animations to demonstrate how the component works.
 
-### Variations (H2)
+### Variations 
 Showcase different forms of your component and include explanations indicating when to use each one. 
 
 Additional sub-sections to include:
-#### When to use X variation vs. when to use Y variation
+#### When to use X variation vs. when to use Y variation (H3)
 
-### Spacing (H2)
+### Spacing 
 Introduce spacing requirements for components and their content. For more information regarding proper spacing, refer to our [spacing guidelines.](https://www.patternfly.org/v4/guidelines/spacers/)
```

Comment:
the "spacing guidelines" link needs to be updated! -- it's now "https://www.patternfly.org/design-foundations/spacers"